custom/icds/templates/icds/messaging/indicators/tam/aww_vhnd_survey.txt

```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+रिकार्ड के जानकारी के आधार पर आापके केन्द्र में पूर्व माह में VHSND नहीं आयोजित किया गया है। कृपया प्रत्येक माह नियमित रूप से VHSND का आयोजन सुनिश्चित करें।
```

Comment:
Hindi?

Reply:
Correc, this is Hindi. I replaced all of the Hindi in the templates with the Tamil translations.

Reply:
@marissahrrsn I must have missed the PR with the updated translation for this one. Can you link me to it?

Reply:
I was reading too quickly. Please ignore my previous comment. Sorry for the noise.